vpc-flow-logs.tf

```diff
@@ -1,3 +1,7 @@
+data "aws_region" "current" {}
```

Comment:
put these behind the create variable - if we aren't creating anything, these APIs shouldn't be called

Reply:
Fixed, thank you.